classes/core/PKPContainer.inc.php

```diff
@@ -215,7 +210,11 @@ protected function loadConfiguration()
             'channel' => 'errorlog',
         ];
 
-        $items['mail']['default'] = static::getDefaultMailer();
+        $items['mail']['default'] = static::getDefaultMailer() === 'disabled' ? 'sendmail' : static::getDefaultMailer();
+        // Replace it by a mocked instance, just to keep things working
+        if (static::getDefaultMailer() === 'disabled') {
+            Mail::fake();
+        }
```

Comment:
Mostly to hide error messages in the build, but perhaps it's useful for an user that doesn't have an email server o_O